src/gl-matrix.js

```diff
@@ -726,9 +722,7 @@ mat4.determinant = function (mat) {
  * Calculates the inverse matrix of a mat4
  *
  * @param {mat4} mat mat4 to calculate inverse of
- * @param {mat4} [dest] mat4 receiving inverse matrix. If not specified result is written to mat
- *
- * @param {mat4} dest is specified, mat otherwise, null if matrix cannot be inverted
+ * @param {mat4} [dest] mat4 receiving inverse matrix. If not specified result is written to mat, null if matrix cannot be inverted
```

Comment:
What are these edits for? `gl-matrix.js` comes from upstream https://github.com/toji/gl-matrix/blob/master/dist/gl-matrix.js, I think we should not do prose changes. Or are these just to accommodate a diagnostic from updated Closure?

Reply:
These are to reduce number of warnings by Closure Compiler. They are not fatal errors though, so I think it is possible to revert changes is needed. Also linked upstream file seems to diverge from this one and doesn't have issues I've fixed here (time to update it?).